packages/kuma-gui/src/app/meshes/routes.ts

```diff
@@ -13,7 +13,7 @@ export const routes = (
 ): RouteRecordRaw[] => {
   return [
     {
-      path: '/meshes',
```

Comment:
These slashes ended up being the main "root" of my issues. Now we are nesting everything within a `/` nothing else should have a `/` in front of them.